schemas/oic.collection-schema.json

```diff
@@ -1,138 +1,139 @@
+            "items": [
+                {
+                    "$ref": "#/definitions/oic.collection.tagged-setoflinks"
+                }
+            ],
+            "additionalItems": false
         },
-        {
-          "properties": {
-            "n": {
-              "type": "string",
-              "description": "User friendly name of the collection",
-              "format": "UTF8"
-            },
-            "id": {
-              "oneOf": [
+        "oic.collection.alllinks": {
+            "description": "All forms of links in a collection",
+            "oneOf": [
                 {
-                  "type": "integer",
-                  "description": "A number that is unique to that collection; like an ordinal number that is not repeated"
+                    "$ref": "#/definitions/oic.collection.setof-tagged-setoflinks"
                 },
                 {
-                  "type": "string",
-                  "description": "A unique string that could be a hash or similarly unique"
+                    "$ref": "#/definitions/oic.collection.tagged-setoflinks"
                 },
                 {
-                  "$ref": "oic.types-schema.json#/definitions/uuid",
-                  "description": "A unique string that could be a UUIDv4"
+                    "$ref": "#/definitions/oic.collection.setoflinks"
+                }
+            ]
+        },
+        "oic.collection": {
+            "type": "object",
+            "description": "A collection is a set (array) of tagged-link or set (array) of simple links along with additional properties to describe the collection itself",
+            "properties": {
+                "n": {
+                    "type": "string",
+                    "description": "User friendly name of the collection"                },
+                "id": {
+                    "anyOf": [
+                        {
+                            "type": "integer",
+                            "description": "A number that is unique to that collection; like an ordinal number that is not repeated"
+                        },
+                        {
+                            "type": "string",
+                            "description": "A unique string that could be a hash or similarly unique"
+                        },
+                        {
+                            "$ref": "#/definitions/uuid",
+                            "description": "A unique string that could be a UUIDv4"
+                        }
+                    ],
+                    "description": "ID for the collection. Can be an value that is unique to the use context or a UUIDv4"
+                },
+                "di": {
+                    "$ref": "#/definitions/uuid",
+                    "description": "The device ID which is an UUIDv4 string; used for backward compatibility with Spec A defintion of /oic/res"
+                },
+                "rts": {
+                    "type": "string",
```

Comment:
This type does not match the example in the oic.wk.col.raml line 76 above, in which the 'rts' value contains an array of strings:
"rts": [ "oic.r.switch.binary", "oic.r.airFlow" ]